.github/workflows/codeql-analysis.yml

```diff
@@ -34,10 +23,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: ["javascript"]
-        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
+        language: ["javascript-typescript"]
```

Comment:
Think this is the source of this
https://github.com/GCTC-NTGC/gc-digital-talent/pull/8761/checks?check_run_id=19522047268

Reply:
Yes, this is a weird one because some of the old workflows will need to be disabled once this PR is merged. I am not exactly sure why but it might be that the CodeQL workflow only works when committed directly to the `main` branch.

> Warning: Code scanning cannot determine the alerts introduced by this pull request, because 1 configuration present on refs/heads/main was not found.

Reply:
this also might need one of the repo @GCTC-NTGC/admin to change any settings: https://docs.github.com/en/code-security/code-scanning/enabling-code-scanning/configuring-default-setup-for-code-scanning#configuring-default-setup-for-a-repository